bitrise.yml

```diff
@@ -81,7 +81,7 @@ workflows:
     before_run:
       - primary
     envs:
-      - GRADLE_TASKS: assembleApp assembleIccaRelease bundleAppUniversal -x assembleAppDebug
```

Comment:
so just to clarify, this is replacing the command that builds the aab with the commands to build the apks instead?

Reply:
Yes, I define the tasks in the environment variable so that I can share the same build code for the different builds (internal, release candidate, and release). The naming of these is `${task}${flavor}${builtType}`. I removed the `bundleAppUniversal` task which would create a `.aab` of the "app" flavor and "universal" build type (the universal build type includes both arm and x86, 32- and 64-bit). I replaced it with the `assembleAppUniversal` task which builds the same but as a `.apk`. I also added a new built type `intel` which only includes the x86 binaries. When publishing multiple APKs to the Play Store they must not overlap (e.g. we could not publish the `release` build (which is only Arm) and the `universal` build, because the `universal` build includes Arm as well as x86.